components/progress/__tests__/index.test.tsx

```diff
@@ -385,4 +385,23 @@ describe('Progress', () => {
     );
     errSpy.mockRestore();
   });
+
+  it('should be accessible', () => {
+    const { container: wrapper, rerender } = render(
+      <Progress percent={70} ariaLabel="My progress" />,
```

Comment:
```suggestion
      <Progress percent={70} aria-label="My progress" />,
```

https://legacy.reactjs.org/docs/accessibility.html#wai-aria